integrations/jquery.md

```diff
@@ -104,6 +109,6 @@ $(document).on('focusin', function(e) {
 
 This code is required because jQuery blocks all `focusin` calls from elements outside the dialog. For a working example, [try this {{site.productname}} fiddle](http://fiddle.tinymce.com/rsdaab/840).
 
-#### A note about integrations
+## A note about integrations
 
-> Note: We are pleased to provide integrations/code guidance to help you build great products with {{site.productname}}. If you have queries about this integration, post your question on Stack Overflow using the [`{{site.prodnamecode}}` tag]({{site.communitysupporturl}}).
+> **Note**: We are pleased to provide integrations/code guidance to help you build great products with {{site.productname}}. If you have queries about this integration, post your question on Stack Overflow using the [`{{site.prodnamecode}}` tag]({{site.communitysupporturl}}).
```

Comment:
Can `{{site.thirdPartyInteg}}` be used here instead?

Reply:
It's not third-party, that's our JQuery integration thingy